server/src/modules/global_data_sources/global_data_sources.module.ts

```diff
@@ -0,0 +1,70 @@
+import { Module } from '@nestjs/common';
```

Comment:
Do we need a new module for global data source?
shall we add the controller to existing data sources module?
/src/modules/data_sources/data_sources.module.ts
```
controllers: [DataSourcesController, GlobalDataSourcesController],
```